virttest/utils_test/libvirt.py

```diff
@@ -787,7 +787,11 @@ def mkfs(partition, fs_type, options="", session=None):
     """
     Make a file system on the partition
     """
-    mkfs_cmd = "mkfs.%s -F %s %s" % (fs_type, partition, options)
+    if fs_type == "xfs":
+        force = "-f"
+    else:
+        force = "-F"
+    mkfs_cmd = "mkfs.%s %s %s %s" % (fs_type, force, partition, options)
```

Comment:
Actually, I wrote a patch which also touch this part long time ago. And maybe pass both fs_type as mkfs options is better, like:
```
+    mkfs_cmd = "mkfs -t %s %s %s" % (fs_type, options, partition)
```